manifests/z/Zen-Team/Zen-Browser/1.0.0-a.31/Zen-Team.Zen-Browser.installer.yaml

```
@@ -0,0 +1,28 @@
+# Created with komac v2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: Zen-Team.Zen-Browser
+PackageVersion: 1.0.0-a.31
+InstallerType: exe

```

The package won't install correctly if this isn't set to `nullsoft`
```suggestion
InstallerType: nullsoft
```